word_counter.rb

```diff
@@ -0,0 +1,38 @@
+
+class Counter
+  def initialize(file)
+    @file = File.open(file)
+    @txt_array = []
+  end
+
+  def display
+    occurence.each do |x,y|
+      puts "#{x}: #{y}"
+    end
+  end
+
+  def parse
+    @file.each do |line|
+      line = line.encode('UTF-8', :invalid => :replace)
```

Comment:
if you wanted to shorten this, you probably can encode on `.open` or `.read` and completely do away with this method.